test/startservers.py

```diff
@@ -32,29 +32,18 @@ def run(self):
 processes = []
 
 
-def install(progs, race_detection):
-    cmd = "go install"
+def install(race_detection):
+    # Pass an empty BUILD_TIME flag to avoid constantly invalidating the build
+    # cache with new BUILD_TIMEs.
+    cmd = "make BUILD_TIME="
     if race_detection:
-        cmd = """go install -race"""
+        cmd = cmd + "GO_BUILD_FLAGS=-race"
 
-    for prog in progs:
-        cmd += " ./" + prog
-    p = subprocess.Popen(cmd, shell=True)
-    out, err = p.communicate()
-    if p.returncode != 0:
-        sys.stderr.write("unable to run go install: %s\n" % cmd)
-        if out:
-            sys.stderr.write("stdout:\n" + out + "\n")
-        if err:
-            sys.stderr.write("stderr: \n" + err + "\n")
-        return False
-    print('installed %s with pid %d' % (cmd, p.pid))
-    return True
+    return subprocess.call(cmd, shell=True) == 0
 
-def run(path, race_detection, config=default_config):
-    binary = os.path.basename(path)
+def run(binary, race_detection, config=default_config):
```

Comment:
this config= seems unnecessary?

Alternatively, it could be just a "command args" or something and we could fit the ocsp-responder in to using run() then.